exonum/src/events/noise/wrappers/sodium_wrapper/wrapper.rs

```diff
@@ -26,8 +26,8 @@ use std::fmt::{self, Error, Formatter};
 use super::{handshake::HandshakeParams, resolver::SodiumResolver};
 use events::noise::{error::NoiseError, HEADER_LENGTH, MAX_MESSAGE_LENGTH, TAG_LENGTH};
 
-pub const HANDSHAKE_HEADER_LENGTH: usize = 1;
-pub const MAX_HANDSHAKE_MESSAGE_LENGTH: usize = 255;
+pub const HANDSHAKE_HEADER_LENGTH: usize = 2;
+pub const MAX_HANDSHAKE_MESSAGE_LENGTH: usize = 65535;
```

Comment:
It seems that Noise does not support messages bigger than this. Maybe we could use a comment here in case somebody wants to expand the header length to 4 bytes some day.